### PR TITLE
Ab#9361 Translations and css availability label live events

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
ADO card: ☑️ [AB#9361](https://dev.azure.com/S72/fefacba9-96b6-4af2-a53d-050ed453e13a/_workitems/edit/9361)

Elab notes/AC: 📃 [Google Docs](https://docs.google.com/document/d/1xVECgmuo5F5HxacMQQUDC3wXsbcCdOQyExJbL28B00Y/edit?pli=1#heading=h.4u4auh6hkf7w)
- [ ] Has this been discussed with Delivery Team?

Designs: 
- 🖥 [mobile/tablet/desktop](https://app.zeplin.io/project/607f961cf9f03790cc66e475/screen/63044e6e8388db11c0f51a35)

## Description of work
All translations needed for live player except the shopping modal blurb

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [ ] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 - Added support for self-service CSS and brand images.
 - Added support for carousel_play_speed and carousel_fade_time configs.
 - Added support to toggle on cloudsearch via Meta > cloudsearch feature toggle.
+- Add translations for live events
```
Comment (PeculiarGoat) [inline CHANGELOG.md:13]: I think you need to merge the latest from develop into your branch... this block of stuff has been released under 1.4.0

**Review by PeculiarGoat — APPROVED** [review]
LGTM apart from that changelog thing!

File: CHANGELOG.md (hunk 1 above)
Comment (matthewthomsonnz) [reply CHANGELOG.md]: mybad moved line to unreleased

**Review by l0ud0gg — APPROVED** [review]
lgtm. just make sure they are also added to db if not already

**Comment by matthewthomsonnz** [comment]
@l0ud0gg @PeculiarGoat Hey guys sorry I'll get you to reapprove. Someone wanted me to put the css from the[ relish branch ](https://github.com/indiereign/shift72-web/pull/344)into the template.

> lgtm. just make sure they are also added to db if not already

@l0ud0gg  Is there a guide?

File: site/styles/_availability-tags.scss [hunk 2]
```diff
@@ -62,6 +62,23 @@ s72-availability-label {
...
+  .availability-bg.live {
+    color: var(--body-color, rgb(255, 255, 255));
+    background-color: rgba(var(--body-bg-rgb, 0, 0, 0), 0.7);
+  }
+
+  s72-availability-status {
+    .availability-livenow:before {
+      content: "";
+      border-radius: 50%;
+      background-color: rgb(var(--poster-live-circle-color-rgb, (255, 0, 0)));
```
Comment (PeculiarGoat) [inline site/styles/_availability-tags.scss:75]: Where is that variable defined?

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Got some merge conflicts to resolve!

File: site/styles/_availability-tags.scss (hunk 2 above)
Comment (matthewthomsonnz) [reply site/styles/_availability-tags.scss]: Changed to `--live-red` my bad

**Comment by matthewthomsonnz** [comment]
> Got some merge conflicts to resolve!

merged in develop

File: site/styles/_variables.scss [hunk 3]
```diff
@@ -51,8 +54,12 @@
   --body-bg-accent: rgb(var(--body-bg-accent-rgb));
   --input-focus-border-color: rgb(var(--input-focus-border-color-rgb));
   --hover-background: rgb(var(--hover-background-rgb));
+  --live-red: rgb(var(--live-red-rgb));
   /*#endregion BASE COLOURS */
 
+  --live-lozenge-background-color: var(--hover-background);
```
Comment (additiverich) [inline site/styles/_variables.scss:60]: We're going with 'label' instead of 'lozenge' fwiw

**Review by additiverich — APPROVED** [review]
lgtm although the reference to 'lozenge' needs to change for my wellbeing if nothing else

**Comment by matthewthomsonnz** [comment]
@PeculiarGoat @additiverich I've made two more small changes:
- poster live availability-status now uses s72-icon-live mixin
- added `display: none` to film detail page poster live availability-status e.g.
image

File: site/styles/_availability-tags.scss [hunk 4]
```diff
@@ -62,6 +62,20 @@ s72-availability-label {
...
     text-align: center;
   }
+
+  .availability-bg.live {
+    color: var(--body-color, rgb(255, 255, 255));
+    background-color: rgba(var(--body-bg-rgb, 0, 0, 0), 0.7);
+  }
+
+  s72-availability-status {
+    .availability-livenow:before {
```
Comment (additiverich) [inline site/styles/_availability-tags.scss:72]: Is the class 'livenow' because we didn't put a dash in before the current status?

Comment (matthewthomsonnz) [reply site/styles/_availability-tags.scss:72]: Nope intentional for consistency see state [here](https://github.com/indiereign/shift72-web/blob/master/src/ui/availability/availability.component.js#L188)

Follows suit with [COMINGSOON](https://github.com/indiereign/shift72-web/blob/master/src/ui/availability/availability.component.js#L166) etc
gets downcased to `class="availability-bg availability-comingsoon"`

**Review by additiverich — APPROVED** [review]

**Pull request merged**